_posts/2020-09-29-becoming-mobile-first-at-artsy.markdown

```diff
@@ -0,0 +1,340 @@
+to the _web_, not the App Store. iOS software is deployed to our user's hardware, not to servers we control, which
+introduces the possibility that users might no upgrade. Software we shipped years ago is still being ran today – we
+have the anayltics to prove it. Not to mention that every app update has to go through Apple's App Store review
+process. Getting our product team aligned on a release schedule might also help us get aligned on the weirdness of
+deploying iOS software.
+
+<center>
+<blockquote class="twitter-tweet"><p lang="en" dir="ltr">iOS developers! I have a question for you. I hear a lot about teams releasing app updates on a 2-week cadence, to increase user confidence/App Store ranking/team morale/etc.<br><br>Have any teams shared their experiences with this? Bonus points for any quantitative data. Thanks!! 🤗</p>&mdash; Ash Furrow (@ashfurrow) <a href="https://twitter.com/ashfurrow/status/1184845989510811649?ref_src=twsrc%5Etfw">October 17, 2019</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+</center><br />
+
+As the Mobile Experience team formed, I reached out to other mobile teams to find out about how they structured
+regular releases. [Matt Greenwell](https://twitter.com/Cform) from BuzzFeed was really helpful in particular,
+outlining their experience of the pros and cons of a regular release cadence. We implemented a two-week release
+cadence so that all product teams could align their own feature development and testing around this predictable
+structure. We also created documentation for teams to hide their in-progress work behind feature flags. And
+finally, we refined our app QA process; teams would QA their own features and bug fixes while the Mobile Experience
+team would used a QA script to test the app _generally_, every other Monday, before submitting an update to the App
+Store.
+
+![Calendar describing our cadence on an example sprint](/images/2020-09-29-becoming-mobile-first-at-artsy/cadence.png)
+
+## Making Deploys Not Scary
+
+To further help teams get into the habit of developing and releasing iOS software often, we created nightly betas.
+We also adopted a clever idea from our web colleagues: deploy blocks. In case of a technical reason to block
+deploys, we create a block and the CI job that deploys the beta would fail with a descriptive message. This reduced
+a lot of chatter in Slack where engineers would ask "could I make a new beta?" Instead, engineers usually just wait
+for the nightly beta. And if they get impatient, they now default to action (their beta deploy will fail if we set
+up a block).
+
+All of this was automated through [fastlane](https://fastlane.tools) on our CI provider. We had been using fastlane
+for a long time at Artsy, but the Mobile Experience team took the time to share knowledge of how it worked. Any
+engineer at Artsy can now make a beta (`make beta`) or promote the latest beta to an App Store submission
```

Comment:
should this be `make deploy`?  Maybe better to update command than blog haha, `make beta` is pretty nice

Reply:
Ah! Yes 😅